CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Changelog
 
 ## Unreleased
+
+## [4.0] - 2021-05-11
```

Comment:
This date format is not correct.

Reply:
@tbugfinder this PR has already been merged. You can provide a PR?